godot-core/src/classes/class_runtime.rs

```diff
@@ -102,6 +102,30 @@ pub(crate) fn ensure_object_inherits(derived: ClassName, base: ClassName, instan
     )
 }
 
+#[cfg(debug_assertions)]
+pub(crate) fn ensure_binding_not_null<T>(binding: sys::GDExtensionClassInstancePtr)
+where
+    T: GodotClass + Bounds<Declarer = bounds::DeclUser>,
+{
+    if !binding.is_null() {
+        return;
+    }
+
+    // Non-runtime classes can't be instantiated in the editor.
+    if crate::classes::Engine::singleton().is_editor_hint() {
+        panic!(
+            "Class {} -- null instance; does the class have a Godot creator function? \
```

Comment:
Newline would be nice:
```suggestion
            "Class {} -- null instance; does the class have a Godot creator function?\n\
```

Reply:
I thought so, but it weirdly splits the warning into two which is a bit too confusing:

![image](https://github.com/user-attachments/assets/50c482d2-5544-4630-90a6-6bf198fab8f2)

Reply:
Hm, but we already use multiple lines in many other error messages. Then we should rather fix how it's represented in Godot error printing (outside this PR), no?